policies/optimized-images.md

```diff
@@ -239,104 +138,94 @@ For an `<img>` element, if neither the width or the height of the source image e
 </br></br>
 
-<a name="image-compression">
+<a name="unoptimized-images">
 
-### "image-compression" policy
+### "unoptimized-images" policy
 
 </a>
 
-When optimizing images, the file size should be kept as small as possible. The larger the download size is, the longer it takes a page to load. Stripping metadata, or using image compression, is a common way to optimize an image's file size. `image-compression` is a policy controlled feature that restricts images to have a file size (in terms of number of bytes) no more than X times bigger than the image size (width * height) on the web page.
+When optimizing images, the file size should be kept as small as possible. The larger the download size is, the longer it takes a page to load. Stripping metadata, picking a good image format, and using image compression, are all common ways to optimize an image's file size. `unoptimized-images` is a policy controlled feature that restricts images to have a file size (in terms of number of bytes) no more than X times bigger than the image size (width * height) on the web page.
 
-When a document is disallowed to use `image-compression` policy, its `<img>` elements whose file sizes are too big will be rendered with inverted colors.
+When a document is disallowed to use `unoptimized-images` policy, its `<img>` elements whose file sizes are too big will be rendered as placeholder images.
 
 
 #### Specification
-
-- The default compression ratio is tentatively 10.
-
-    **Note**: We want to allow developers the ability to make the final decision about the tradeoffs they make. The goal is to eventually introduce a syntax for specifying their own ratio.
+- The default maximum file size of an optimized image is calculated as following:
+    
+   ```metadata size limit + byte-per-pixel ratio * image resolution```
+    + For images of one of the modern formates (JPEG, PNG, GIF, WEBP, and SVG)
+        + The default metadata size limit is tentatively 1KB.
+        + The default byte-per-pixel ratio is tentatively 0.5.
+    + For images of other legacy formats   
+        + The metadata size limit is set to 0KB.
+        + The byte-per-pixel ratio is set to 0.
+
+    **Note**: We want to allow developers the ability to make the final decision about the tradeoffs they make. The goal is to eventually introduce a syntax for specifying their own parameters.
 
     In practice, they would look something like this:
 
     ```html
-    <iframe allow="image-compression(12)"></iframe>
-    ```
-    That would apply a policy in which the maximum compression ratio allowed is set to 12.
-
-    Feature policies combine in subframes, and the minimum value of the compression ratio will be applied, so if that frame embedded another, which the syntax:
-
-    ```html
-    <iframe allow="maximum-downscaling-image(15)"></iframe>
+    <iframe allow="unoptimized-images(1.5, 0.4)"></iframe>
```

Comment:
Did we decide somewhere to use KB as the unit? Would it be more natural to specify # bytes instead?
(I'm not sure, for instance, whether 1KB = 1000 bytes or 1024, or if 0.2 would mean 200 bytes, or 204, or even 205 if rounded up)

Reply:
makes sense. I used 1024 but yea we should be using bytes.